_episodes/03-ttree-details.md

```diff
@@ -161,10 +161,8 @@ for muons in tree.iterate(filter_name=["nMuon", "/Muon_(pt|eta|phi)/"]):
     pt0  = muons["Muon_pt",  cut, 0]; pt1  = muons["Muon_pt",  cut, 1]
     eta0 = muons["Muon_eta", cut, 0]; eta1 = muons["Muon_eta", cut, 1]
     phi0 = muons["Muon_phi", cut, 0]; phi1 = muons["Muon_phi", cut, 1]
-
```

Comment:
somewhat opinionated, but ok ;)

Reply:
ah, just saw your comment about the broken indentation. Interesting, that shouldn't happen though.